package.json

```diff
@@ -175,6 +175,7 @@
 		"test:php": "node ./tools/local-env/scripts/docker.js run -T php composer update -W && node ./tools/local-env/scripts/docker.js run php ./vendor/bin/phpunit",
 		"test:e2e": "node ./tests/e2e/run-tests.js",
 		"test:visual": "node ./tests/visual-regression/run-tests.js",
-		"sync-gutenberg-packages": "grunt sync-gutenberg-packages --dev"
+		"sync-gutenberg-packages": "grunt sync-gutenberg-packages",
+		"postsync-gutenberg-packages": "grunt wp-packages:sync-stable-blocks && grunt build --dev"
```

Comment:
This is the simplest way to retain the ability to pass dist-tag to `sync-gutenberg-packages` and resolve the issue that blocks need to be regenerated using the dynamically loaded webpack config after the update of all packages.

The best path moving forward would be to completely rebuild the way we handle block assets in `CopyWebpackPlugin`. However, it's rather a big refactoring that would only impact this tool that is going to be executed a few times a year.